src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_blob_live_scenarios.py

```diff
@@ -5,45 +5,54 @@
 
 import os
 from datetime import datetime, timedelta
+
+import pytest
 from azure.cli.testsdk import (LiveScenarioTest, ResourceGroupPreparer, StorageAccountPreparer,
                                JMESPathCheck, JMESPathCheckExists, NoneCheck, api_version_constraint)
 from azure.cli.core.profiles import ResourceType
 
 
 @api_version_constraint(ResourceType.MGMT_STORAGE, min_api='2016-12-01')
 class StorageBlobUploadLiveTests(LiveScenarioTest):
+    @pytest.mark.serial()
```

Comment:
May I know the motivation for the change?

Reply:
Check your mail box, Important update of "User Manual of Live Test Pipeline"